3_Implementation/documentation/html/search/functions_5.js

```diff
@@ -0,0 +1,34 @@
+  ['unityassertnumberswithin_848',['UnityAssertNumbersWithin',['../unity_8c.html#a1edf5167aa7021aba92c523328e8fa61',1,'UnityAssertNumbersWithin(const UNITY_UINT delta, const UNITY_INT expected, const UNITY_INT actual, const char *msg, const UNITY_LINE_TYPE lineNumber, const UNITY_DISPLAY_STYLE_T style):&#160;unity.c'],['../unity__internals_8h.html#a1edf5167aa7021aba92c523328e8fa61',1,'UnityAssertNumbersWithin(const UNITY_UINT delta, const UNITY_INT expected, const UNITY_INT actual, const char *msg, const UNITY_LINE_TYPE lineNumber, const UNITY_DISPLAY_STYLE_T style):&#160;unity.c']]],
+  ['unitybegin_849',['UnityBegin',['../unity_8c.html#aed50601fbdaea8fc0fef0c61cf877809',1,'UnityBegin(const char *filename):&#160;unity.c'],['../unity__internals_8h.html#aed50601fbdaea8fc0fef0c61cf877809',1,'UnityBegin(const char *filename):&#160;unity.c']]],
+  ['unityconcludetest_850',['UnityConcludeTest',['../unity_8c.html#a44f03f0124e59f6996da61946cc2a58e',1,'UnityConcludeTest(void):&#160;unity.c'],['../unity__internals_8h.html#a44f03f0124e59f6996da61946cc2a58e',1,'UnityConcludeTest(void):&#160;unity.c']]],
+  ['unitydefaulttestrun_851',['UnityDefaultTestRun',['../unity_8c.html#ae3c8272eef0f0b6228ca47b9dd1aa2af',1,'UnityDefaultTestRun(UnityTestFunction Func, const char *FuncName, const int FuncLineNum):&#160;unity.c'],['../unity__internals_8h.html#ae3c8272eef0f0b6228ca47b9dd1aa2af',1,'UnityDefaultTestRun(UnityTestFunction Func, const char *FuncName, const int FuncLineNum):&#160;unity.c']]],
+  ['unityend_852',['UnityEnd',['../unity_8c.html#a4820ba04ff5e9a474fe38ab6633d30a0',1,'UnityEnd(void):&#160;unity.c'],['../unity__internals_8h.html#a4820ba04ff5e9a474fe38ab6633d30a0',1,'UnityEnd(void):&#160;unity.c']]],
+  ['unityfail_853',['UnityFail',['../unity_8c.html#a0daba95a4950d8a5a06b6b2c843742b8',1,'UnityFail(const char *msg, const UNITY_LINE_TYPE line):&#160;unity.c'],['../unity__internals_8h.html#a52861126192e5b2162adcf68bae8d213',1,'UnityFail(const char *message, const UNITY_LINE_TYPE line):&#160;unity.c']]],
+  ['unityfloattoptr_854',['UnityFloatToPtr',['../unity_8c.html#a01b1539781210d836d875fa9c6957982',1,'UnityFloatToPtr(const float num):&#160;unity.c'],['../unity__internals_8h.html#a01b1539781210d836d875fa9c6957982',1,'UnityFloatToPtr(const float num):&#160;unity.c']]],
+  ['unityignore_855',['UnityIgnore',['../unity_8c.html#a017e8c6a617d00312fd6688b73483284',1,'UnityIgnore(const char *msg, const UNITY_LINE_TYPE line):&#160;unity.c'],['../unity__internals_8h.html#a26d523163b74734d8cbb1413e687edf6',1,'UnityIgnore(const char *message, const UNITY_LINE_TYPE line):&#160;unity.c']]],
+  ['unitymessage_856',['UnityMessage',['../unity_8c.html#abc9ecca68af9f93f07fb907de67d5db3',1,'UnityMessage(const char *msg, const UNITY_LINE_TYPE line):&#160;unity.c'],['../unity__internals_8h.html#aab07c6775c63f8976893ab8cd8e5d14c',1,'UnityMessage(const char *message, const UNITY_LINE_TYPE line):&#160;unity.c']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unitymessage_856",['UnityMessage',['../unity_8c.html#abc9ecca68af9f93f07fb907de67d5db3',1,'UnityMessage(const char *msg, const UNITY_LINE_TYPE line):&#160;unity.c'],['../unity__internals_8h.html#aab07c6775c63f8976893ab8cd8e5d14c',1,'UnityMessage(const char *message, const UNITY_LINE_TYPE line):&#160;unity.c']]],
```